src/hotspot/cpu/x86/x86_64.ad

```diff
@@ -5693,6 +5801,19 @@ instruct countTrailingZerosL_mem(rRegI dst, memory src, rFlagsReg cr) %{
   ins_pipe(ialu_reg_mem);
 %}
 
+instruct countTrailingZerosL_mem_nf(rRegI dst, memory src, rFlagsReg cr) %{
```

Comment:
Why are we using rflagsReg operand for Non-Flags affecting patterns?

Reply:
Thanks for the catch! Please see the updated NF code for count instructions which reverted to the previous implementation which does not use the CR flags.